context/app/static/js/components/organ/CellPop/CellPopDescription.tsx

```diff
@@ -39,7 +39,8 @@ export default function CellPopDescription() {
         </LabelledSectionText>,
       ]}
     >
-      This interactive heatmap visualizes cell populations in datasets from this organ. Cell type annotations are from{' '}
+      The cell population plot includes all RNAseq {organ.toLowerCase()} datasets that contain cell type annotations.
```

Comment:
@tsliaw I added 'RNAseq' here because we still need to limit the cell pop datasets due to scaling issues. Does this description work?

Reply:
Yes that description works.